cfbs/commands.py

```diff
@@ -98,34 +97,28 @@ def pretty_command(filenames: list, check: bool, keep_order: bool) -> int:
 
     cfbs_sorting_rules = None
     if not keep_order:
-        top_level_keys = ("name", "description", "type", "index")
-        module_keys = (
-            "name",
-            "description",
-            "tags",
-            "repo",
-            "by",
-            "version",
-            "commit",
-            "subdirectory",
-            "dependencies",
-            "steps",
+        # These sorting rules achieve 3 things:
```

Comment:
You could do an interactive rebase and squash the typo-fix with the commit that introduced it, hiding the fact that you are only human 😉

Reply:
Done ;)